docs/scripts/util/getFrontmatter.ts

```diff
@@ -0,0 +1,13 @@
+export function getFrontmatter(fileData: string) {
+  // get MDN URL
+  const mdn = fileData.indexOf('MDN:');
+  const endOfLine1 = fileData.indexOf('\n', mdn);
+  const MDN = fileData.slice(mdn, endOfLine1).split(' ')[1];
+
+  // get htmlElement
```

Comment:
nit: how about making a function `getFrontmatterElement(element)` and the `element` can be `mdn`, `htmlElement`, etc, so that we can re-use it?

Reply:
Or, we can consider what we did for `preval`, just grab all the frontmatter elements, so that we don't have to keep modifying this function in the future if we need to grab more than `MDN` and `htmlElement`. Does that make sense?

Reply:
Yes, that makes sense. I'll modify the function to grab all the frontmatter elements